lib/util/metrics.js

```diff
@@ -0,0 +1,70 @@
+let metric = {
+  newConnectionsCount: 0,
+  reconnectionCount: 0,
+  closedConnectionCount: 0,
+  messagesCount: 0,
```

Comment:
a very highly unlikely thing in the near future, but possible when the scale increases that this value can exceed the max integer value of JS.
Just FYI...

Reply:
I think this can be bottleneck it can go beyond int values